01-Introducing-Molecular-Dynamics/02b-Running-Rigid-Body-Molecular-Dynamics-System.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    import rowan
</pre><p>Again, I don't think you use <code>rowan</code>  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>